samples/import_dataset.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+function main(
+  projectId = 'YOUR_PROJECT_ID',
+  datasetId = 'YOUR_DISPLAY_ID',
+  path = 'gs://BUCKET_ID/path_to_training_data.csv'
```

Comment:
Is this something that should be constructed from two separate parameters? Or, do we know always what one of them will be? (Maybe not - lmk what you think)

Reply:
This can be whatever the user wants.